### ISSUE TITLE
New rotation scripts: "Partitioning can not be used stand-alone"

### ISSUE THREAD

**Issue by dougbtv** [sipcapture/homer-api#81]
I haven't dug into this to try to submit a PR, but in trying to implement the new rotation scripts I ran into this error message

```
Partitioning can not be used stand-alone in query near 'PARTITION p2016042500 VALUES LESS THAN (1461628800)' at line 1
```

Full output from running script:

```
root@f10c4691fbd2:/# /opt/homer_rotate/homer_rotate 
ZZ: [user]
ZZ: [password]
ZZ: [host]
ZZ: [port]
ZZ: [db_data]
ZZ: [db_stats]
ZZ: [newtables]
ZZ: [engine]
ZZ: [compress]
ZZ: [sip_capture_call]
ZZ: [sip_capture_registration]
ZZ: [sip_capture_rest]
ZZ: [sip_capture_call]
ZZ: [sip_capture_registration]
ZZ: [sip_capture_rest]
ZZ: [rtcp_capture]
ZZ: [logs_capture]
ZZ: [report_capture]
ZZ: [stats_ip]
ZZ: [stats_geo]
ZZ: [alarm_data]
ZZ: [stats_method]
ZZ: [stats_useragent]
ZZ: [rtcp_capture]
ZZ: [logs_capture]
ZZ: [report_capture]
ZZ: [stats_ip]
ZZ: [stats_geo]
ZZ: [alarm_data]
ZZ: [stats_method]
ZZ: [stats_useragent]
DBD::mysql::db do failed: Partitioning can not be used stand-alone in query near 'PARTITION p2016042500 VALUES LESS THAN (1461628800)' at line 1 at /opt/homer_rotate/homer_mysql_rotate.pl line 345.
Failed to execute query [Partitioning can not be used stand-alone in query near 'PARTITION p2016042500 VALUES LESS THAN (1461628800)' at line 1] with error: DBD::mysql::db do failed: Partitioning can not be used stand-alone in query near 'PARTITION p2016042600 VALUES LESS THAN (1461715200)' at line 1 at /opt/homer_rotate/homer_mysql_rotate.pl line 345.
Failed to execute query [Partitioning can not be used stand-alone in query near 'PARTITION p2016042600 VALUES LESS THAN (1461715200)' at line 1] with error: DBD::mysql::db do failed: Partitioning can not be used stand-alone in query near 'PARTITION p2016042700 VALUES LESS THAN (1461801600)' at line 1 at /opt/homer_rotate/homer_mysql_rotate.pl line 345.
Failed to execute query [Partitioning can not be used stand-alone in query near 'PARTITION p2016042700 VALUES LESS THAN (1461801600)' at line 1] with error: DBD::mysql::db do failed: Partitioning can not be used stand-alone in query near 'PARTITION p2016042500 VALUES LESS THAN (1461628800)' at line 1 at /opt/homer_rotate/homer_mysql_rotate.pl line 345.
Failed to execute query [Partitioning can not be used stand-alone in query near 'PARTITION p2016042500 VALUES LESS THAN (1461628800)' at line 1] with error: DBD::mysql::db do failed: Partitioning can not be used stand-alone in query near 'PARTITION p2016042600 VALUES LESS THAN (1461715200)' at line 1 at /opt/homer_rotate/homer_mysql_rotate.pl line 345.
Failed to execute query [Partitioning can not be used stand-alone in query near 'PARTITION p2016042600 VALUES LESS THAN (1461715200)' at line 1] with error: DBD::mysql::db do failed: Partitioning can not be used stand-alone in query near 'PARTITION p2016042700 VALUES LESS THAN (1461801600)' at line 1 at /opt/homer_rotate/homer_mysql_rotate.pl line 345.
Failed to execute query [Partitioning can not be used stand-alone in query near 'PARTITION p2016042700 VALUES LESS THAN (1461801600)' at line 1] with error: DBD::mysql::db do failed: Partitioning can not be used stand-alone in query near 'PARTITION p2016042500 VALUES LESS THAN (1461628800)' at line 1 at /opt/homer_rotate/homer_mysql_rotate.pl line 345.
Failed to execute query [Partitioning can not be used stand-alone in query near 'PARTITION p2016042500 VALUES LESS THAN (1461628800)' at line 1] with error: DBD::mysql::db do failed: Partitioning can not be used stand-alone in query near 'PARTITION p2016042600 VALUES LESS THAN (1461715200)' at line 1 at /opt/homer_rotate/homer_mysql_rotate.pl line 345.
Failed to execute query [Partitioning can not be used stand-alone in query near 'PARTITION p2016042600 VALUES LESS THAN (1461715200)' at line 1] with error: DBD::mysql::db do failed: Partitioning can not be used stand-alone in query near 'PARTITION p2016042700 VALUES LESS THAN (1461801600)' at line 1 at /opt/homer_rotate/homer_mysql_rotate.pl line 345.
Failed to execute query [Partitioning can not be used stand-alone in query near 'PARTITION p2016042700 VALUES LESS THAN (1461801600)' at line 1] with error: Can't use an undefined value as a HASH reference at /opt/homer_rotate/homer_mysql_rotate.pl line 187.
```

I also have  PR I'm getting ready that addresses the default path for the ini, I'll have it right up.

**Comment by dougbtv** [comment]
Oh yeah script in question is: `scripts/new/homer_mysql_rotate.pl`

**Comment by adubovikov** [comment]
https://github.com/sipcapture/homer-api/commit/a7cf060c10cb5e723a40027f50a2d5cdd97e1467

fixed. Please check it.

**Comment by adubovikov** [comment]
thank you! let me know if get any issues

**Comment by dougbtv** [comment]
Thanks for the quick fix Alexandr! Your fix completely fixed that issue, but, I wound up running into another, I submitted the fix for it in PR #83

Basically what was happening is the script was creating the new tables for today + how many days as set in the ini `newtables` configuration option, but.... It'd basically try to do the same for the `drop table` as well, so it'd drop the tables with today's day (plus yesterday's table). So I tried to fix it to use the horizons from the ini file.
